app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragment.kt

```diff
@@ -312,6 +330,17 @@ class ResourcesFragment(val root: Path?, val path: Path?): MvpAppCompatFragment(
         updateVerticalBias(iv_drag_handler)
     }
 
+    private fun sendShortToastMessage(message: String){
+        Toast.makeText(requireContext(), message, Toast.LENGTH_SHORT).show()
+    }
+
+    private fun changeSortOrderLockStatus(dialogView: View, isEnabledStatus: Boolean){
```

Comment:
Just a question: what is "lock" here?

Reply:
I guess I wasn't able to come up with an elegant explanation of the method in its name :)
`changeSortOrderLockStatus` enables or disables "Ascending" and "Descending" radio buttons according to the passed value. I just wrapped it in one method for convenience.